Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -322,7 +328,7 @@ private async Task CheckMultipleSelectionSetEmpty( ParameterView parameters )
 
         private async Task HandleSelectionModeChanged()
         {
-            if ( selectionMode == DataGridSelectionMode.Multiple && SelectedRow != null )
+            if ( SelectionMode == DataGridSelectionMode.Multiple && SelectedRow != null )

```

Comment:
Use `is not null` instead of `!= null`? I have also started to use it more lately.

Reply:
Right. Its not new code. Sometimes I forget to change. We should do a wide PR that changes this in one of the next releases.

Reply:
Yeah, we can clean this after or for 1.2.